src/Conf.ml

```diff
@@ -1766,9 +1760,11 @@ let rec collect_files ~segs ~ignores ~enables ~files =
         if Fpath.exists filename then filename :: enables else enables
       in
       let files =
-        let filename = Fpath.(dir / dot_ocp_indent) in
-        if Fpath.exists filename then `Ocp_indent filename :: files
-        else files
+        let f_1 = Fpath.(dir / dot_ocamlformat) in
+        let f_2 = Fpath.(dir / dot_ocp_indent) in
+        files
+        |> (fun k -> if Fpath.exists f_1 then `Ocamlformat f_1 :: k else k)
```

Comment:
I think ``let files = if Fpath.exists f_1 then `Ocamlformat f_1 :: files else files in`` would be a little more readable. That way `f_1` and `f_2` could be defined just before they are used.

Reply:
done